tools/protoprint/protoprint.py

```diff
@@ -17,8 +16,16 @@
 import re
 import subprocess
 import sys
+from collections import deque
+from functools import cached_property
+
+from packaging import version
+
+from bazel_tools.tools.python.runfiles import runfiles
 
-from tools.api_proto_plugin import annotations, traverse, visitor
+sys.path = [p for p in sys.path if not p.endswith('bazel_tools')]
```

Comment:
Seems a bit evil..

Reply:
this is the usual hack, as added elsewhere - if you use `runfiles` you cant import tools without mangling `sys.path`

Reply:
added here because i added `runfiles` to get the clang-tidy config - otherwise i generally avoid runfiles if i can

Reply:
Can you factor this pattern out into some library so it's clear what is going on?

Reply:
not really - its one line and needs to be explicit

this is really nothing new - you and i have added a few of these - and i have removed them whenever i can

Reply:
ill add a comment to make it clearer